packages/pds/src/api/com/atproto/server/createAccount.ts

```diff
@@ -101,6 +95,7 @@ export default function (server: Server, ctx: AppContext) {
         throw err
       }
 
+      // insert invite code use & do an integrity check to make sure we didn't over-use it
```

Comment:
Based on this comment and the PR description, I sort of thought I'd see some kind of `ensureCodeIsAvailable()` check after the `invite_code_use` is created.  But at the same time, the way it's currently written seems perfectly fine to me!  Did the approach change a little bit since this comment was written? Or maybe I'm just misinterpreting it!

Reply:
ah the comment was from when i _did_ have that exact check. but I ended up rearranging it 👍